.gitignore

```diff
@@ -0,0 +1,4 @@
+Data_Output/
+Data_Input/
+venv/
+.idea/
```

Comment:
Please add newline at the end of the file. A lot of unix tools rely on \n symbol and it is considered a good practice to have a new line at the end of each file.